src/datasets/commands/dummy_data.py

```diff
@@ -295,14 +295,9 @@ def run(self):
         auto_generate_results = []
         with tempfile.TemporaryDirectory() as tmp_dir:
             for builder_config in builder_configs:
-                if builder_config is None:
-                    name = None
-                    version = builder_cls.VERSION
-                else:
-                    version = builder_config.version
-                    name = builder_config.name
-
+                name = builder_config.name if builder_config else None
                 dataset_builder = builder_cls(name=name, hash=dataset_module.hash, cache_dir=tmp_dir)
+                version = builder_config.version if builder_config else dataset_builder.config.version
```

Comment:
this is maybe simpler ?
```suggestion
                version = dataset_builder.config.version
```